portfolio/src/main/webapp/script.js

```diff
@@ -324,3 +353,39 @@ function initializeMap() {
   });
   initializeMarkers(map);
 }
+
+/**
+ * Retrieves latitude and longitude values from respective fields and queries
+ * the Google Geocoding API for an approximate address string. If an error occurs
+ * in the API fetch or bad input is provided, error message is alerted.
+ */
+async function getAddress() {
+  const lat = document.getElementById("lat").value;
+  const lng = document.getElementById("lng").value;
+  if (
+    lat > 90 ||
+    lat < -90 ||
+    lng > 180 ||
+    lng < -180 ||
+    lat.length === 0 ||
+    lng.length === 0
+  ) {
+    alert("Enter a valid latitude/longitude pair!");
```

Comment:
Can you limit the range of lat lng in the "input" tag instead of popping out an alert here?

Reply:
I just added `min` and `max` attributes to the `<input>` tags, but they can't be validated with a 
 native popup like this 
![image](https://user-images.githubusercontent.com/7517829/84214549-abf28900-aa91-11ea-9fef-cc4a073b3e26.png) 
since the Check Address button is not of input type `= submit` and I prevented its default behavior to prevent it from redirecting to any URI on click (line 212 of script.js in latest commit). Now, the alert window appears when the lat/lng are out of range but with the error message from the API. Should I keep this behavior?

Reply:
It seems that vanilla JS has no workaround for this to simulate the native popup that Chrome gives me for the form validation. At the very least, the `min` and `max` attributes in the input tag should help with accessibility.

Reply:
Ahh okay, yeah, I didn't notice the button is not a submit. Leave it be then, I don't think it is worth spending much time on vanilla form.